frontend/src/profile/Profile.tsx

```diff
@@ -140,23 +136,23 @@ interface ChangePasswordProps {
   id: number;
 }
 
-const ProfileName = (props: ProfileNameProps) => {
-  return (
-    <ProfileEntryContainer>
-      <ItemTitle> Name </ItemTitle>
-      {props.isEdit && (
-        <TextField
-          id="outlined-basic"
-          variant="outlined"
-          value={props.name}
-          onChange={e => props.setName(e.target.value)}
-          placeholder="John Smith"
-        />
-      )}
-      {!props.isEdit && <ItemEntry> {props.name} </ItemEntry>}
-    </ProfileEntryContainer>
-  );
-};
+// const ProfileName = (props: ProfileNameProps) => {
```

Comment:
Going to re-implement this in a separate PR, as right now we don't even show the Profile page. Right now we want to show their full name, but not allow them to edit it.

Reply:
yea we might want to rethink the UI/purpose of this page, probably not super high priority right now